apps/mitt-konto/src/MittKonto/Main.purs

```diff
@@ -163,6 +164,12 @@ userView { user } = React.fragment
   , classy DOM.div "col col-12 md-col-6 lg-col-6" [ subscriptionsView ]
   ]
   where
+    subsSorted = foldl canceledLast [] user.subs
+
+    canceledLast subs s
```

Comment:
Note that this reverses the order of active subscriptions. If the subscriptions are in no particular order coming from Persona, this works fine, otherwise, this should be done in some other manner.

Reply:
I wonder if we should just define an `Ord` for `SubscriptionState` so we can `sort`?